chart/values.yaml

```diff
@@ -55,6 +55,11 @@ serviceAccount:
     podRun: true
     # exec
     exec: true
+
+impersonationRole:
```

Comment:
Instead of this key, should we use
```yaml
serviceAccount:
  rbac:
    impersonate:
      enabled: boo
      createNamespaces: bool
      namespaces: list[string]
```